tests/user_restricted/conftest.py

```diff
@@ -1,24 +1,93 @@
 import pytest
 
 from .utils import helpers
-from .utils.check_oauth import CheckOauth
 import uuid
 import random
 from ..scripts import config
+from tests.functional.config_files import config as functional_config
+from pytest_nhsd_apim.apigee_apis import (
+    ApigeeClient,
+    ApigeeNonProdCredentials,
+    ApiProductsAPI,
+    DeveloperAppsAPI,
+)
+import logging
+
+LOGGER = logging.getLogger(__name__)
+
+AUTH_HEALTHCARE_WORKER = {
+    "access": "healthcare_worker",
+    "level": "aal3",
+    "login_form": {"username": "656005750104"},
+}
+DEVELOPER_EMAIL = "apm-testing-internal-dev@nhs.net"
```

Comment:
This is defined in a few places. Is it worth placing somewhere common?

Reply:
The user retricted conftest should go out of use once we move the tests over to use feature files under 1680, leaving just 1 place that defines it. Might be worth getting it from an environment variable, or even better, from a fixture from the new library.